packages/r/rttr/xmake.lua

```diff
@@ -0,0 +1,35 @@
+package("rttr")
+    set_kind("shared")
+    set_homepage("https://www.rttr.org")
+    set_description("rttr: An open source library, which adds reflection to C++.")
+
+    if is_plat("macosx") then
+        add_extsources("brew::rttr")
+    end
+
+    if is_plat("windows") then
+        add_urls("https://www.rttr.org/releases/rttr-$(version)-src.zip",
+                 "https://github.com/rttrorg/rttr/releases/download/v$(version)/rttr-$(version)-src.zip",
+                 "https://github.com/rttrorg/rttr.git")
+
+        -- windows version use \r\n
+        add_versions("0.9.6", "d6853be1bfc38da6fb4512ae4c0a116662f86fa81a6b2aab1c4dada2c16f375f")
+        add_versions("0.9.5", "bab810d99d7c8191fde0032061d6120e4bc99eebb239036e2adaf34ebe92a51f")
+    else
+        add_urls("https://www.rttr.org/releases/rttr-$(version)-src.tar.gz",
+                 "https://github.com/rttrorg/rttr/releases/download/v$(version)/rttr-$(version)-src.tar.gz",
+                 "https://github.com/rttrorg/rttr.git")
+
+        -- unix version use \n
+        add_versions("0.9.6", "f62caee43016489320f8a69145c9208cddd72e451ea95618bc26a49a4cd6c990")
+        add_versions("0.9.5", "caa8d404840b0e156f869a947e475b09f7b602ab53c290271f40ce028c8d7d91")
+    end
```

Comment:
why? please use same url.

it should work for `\n` for win/linux

Reply:
The package maintainer offeres different versions for windows and unix. I'll modify this later.